R/join_keys.R

```diff
@@ -0,0 +1,155 @@
+#' Manage relationships between datasets using `join_keys`
+#'
+#' @description
```

Comment:
I think this should be in Details and what is now in Details should be in Value.

Also, describe class before describing function?